modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -598,7 +553,7 @@ class Instrument_Manager extends \NDB_Menu_Filter
      *
      * @return bool True if a adminUser is configured properly. False if not.
      */
-    protected function isQuatUserConfigured() : bool
+    protected function isAdminUserConfigured() : bool
```

Comment:
This function signature was updated because `quatUser` was renamed to `adminUser`( #5785 )

Reply:
@johnsaigle I missed this!